src/layout.js

```diff
@@ -138,7 +138,7 @@ EPUBJS.Layout.Fixed.prototype.format = function(documentElement, _width, _height
 	*/
 	if(viewport && viewport.hasAttribute("content")) {
 		content = viewport.getAttribute("content");
-		contents = content.split(',');
+		contents = content.split(/\s*,\s*/);
```

Comment:
Added in view_managers: https://github.com/futurepress/epub.js/blob/view_managers/src/contents.js#L161